pkg/segment/tracing/handler/tracehandler.go

```diff
@@ -365,3 +365,83 @@ func redMetricsToJson(redMetrics structs.RedMetrics, service string) ([]byte, er
 	result["p99"] = redMetrics.P99
 	return json.Marshal(result)
 }
+
+func DependencyGraphThread() {
+	time.Sleep(1 * time.Minute) // Wait for initial traces ingest first
+	for {
+		MakeTracesDependancyGraph()
+		time.Sleep(1 * time.Hour)
+	}
```

Comment:
Can you change this so that it computes the graph the first time after one minute (like you have it), but then every time after that it computes the graph at the top of the hour?